### ISSUE TITLE
1.7.10 Version not working at all

### ISSUE THREAD

**Issue by MagmaSlime123** [djpadbit/Sound-Physics#78]
As I explained in #74, the mod itself just outright won't work. It's loaded, but there's 0 reverbing/echoes of any kind, as if the mod isn't installed. This is mainly due to SP having 0 compat./patch with Gibly's Proximity Voice Chat on that version.

Doubt you support 1.7.10 these days, but it does still have a somewhat active-ish modding community that may benefit from this great mod, and it'd be nice if you were able to fix it up and such.

**Comment by makamys** [comment]
Does this older build work? It does for me. https://github.com/djpadbit/Sound-Physics/issues/26#issuecomment-614747542

Actually, I just tried with the latest build in https://github.com/djpadbit/Sound-Physics/issues/74#issuecomment-896336358, and it also works. So it must be an incompatibility with one of your mods.

**Comment by MagmaSlime123** [comment]
I did some testing, Gibly's Proximity Voice Chat is the issue.

Not sure why.

Just using Gibly's Proximity Voice Chat. This mod was updated to 1.12.2, but has since then been dead. So its up to the dev of SP to do something for both 1.7.10 and 1.12.2.

Doubt 1.7.10 is supported these days, but it does still have a somewhat active-ish modding community that may benefit from this great mod.

**Comment by makamys** [comment]
So it's the same issue as #74, just for 1.7.10?

**Comment by MagmaSlime123** [comment]
Yep.

**Comment by jtjjjj08** [comment]
Does the 1.7.10 version still not work with gibbies voice chat?

**Comment by MagmaSlime123** [comment]
Nope. Like I said, its up to the dev of SP to do something for both 1.7.10 and 1.12.2.

**Comment by jtjjjj08** [comment]
I don't think the dev is updating it anymore. Where do you even get the 1.7.10 builds of sound physics> Nope. Like I said, its up to the dev of SP to do something for both 1.7.10 and 1.12.2.
